Library/Homebrew/extend/ENV.rb

```diff
@@ -29,12 +29,16 @@ def with_build_environment
     replace(old_env)
   end
 
-  def clear_sensitive_environment!
-    each_key do |key|
-      next unless /(cookie|key|token|password)/i =~ key
+  def sensitive?(key)
+    /(cookie|key|token|password)/i =~ key
+  end
 
-      delete key
-    end
+  def sensitive_environment
+    select { |key, _value| sensitive?(key) }
```

Comment:
Can be just `|key,|` I think or if not then `|key, _|`

Reply:
Updated